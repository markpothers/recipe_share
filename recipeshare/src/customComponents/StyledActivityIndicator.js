import React from 'react'
import { View, ActivityIndicator, Platform } from 'react-native';
import { centralStyles } from '../centralStyleSheet' //eslint-disable-line no-unused-vars

export default function StyledActivityIndicator() {


	return (
		<View style={centralStyles.activityIndicatorContainer}>
			<ActivityIndicator
				style={(Platform.OS === 'ios' ? centralStyles.activityIndicator : {})}
				size="large"
				color="#104e01"
			/>
		</View>
	)
}
