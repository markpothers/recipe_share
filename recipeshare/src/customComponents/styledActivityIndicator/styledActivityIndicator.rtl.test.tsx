import React from "react"
import { render } from "@testing-library/react-native"
import StyledActivityIndicator from "./styledActivityIndicator"

describe("StyledActivityIndicator", () => {

	test("can be rendered", async () => {
		const { toJSON } = render(<StyledActivityIndicator />)
		expect(toJSON()).toMatchSnapshot()
	})

})
