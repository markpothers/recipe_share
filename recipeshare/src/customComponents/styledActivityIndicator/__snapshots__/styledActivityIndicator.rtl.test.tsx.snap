// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledActivityIndicator can be rendered 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff59b",
      "borderColor": "#104e01",
      "borderRadius": 11.25,
      "borderWidth": 1,
      "height": 50,
      "justifyContent": "center",
      "left": "50%",
      "marginLeft": -25,
      "position": "absolute",
      "top": "10%",
      "width": 50,
      "zIndex": 5,
    }
  }
>
  <ActivityIndicator
    color="#104e01"
    size="large"
    style={
      Object {
        "left": 1.25,
        "top": 1.7,
      }
    }
    testID="activityIndicator"
  />
</View>
`;
